src/main/java/org/springframework/data/tarantool/core/TarantoolTemplate.java

```diff
@@ -88,6 +99,67 @@ public TarantoolTemplate(
         );
     }
 
+    private <T> CallResultMapper<T, SingleValueCallResult<T>>
+    getDefaultWithTupleResultMapper(Class<T> entityClass, Optional<TarantoolSpaceMetadata> spaceMetadata) {
```

Comment:
It seems worth splitting this function into two: `getDefaultResultMapper` and `addTupleResultMapper`

Reply:
+

Reply:
Splited on `getDefaultComplexTypesMapper`, `registerTupleResultMapper`, `withDefaultSingleValueMapper` and `withDefaultSingleValueMapper`